challenges/objects-arrays.js

```diff
@@ -55,11 +79,19 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for (let i = 0; i < graduates.length; i++) {
+  contactInfo.push(`${graduates[i].first_name} ${graduates[i].email}`);
```

Comment:
Love that you are using string literals instead of ES5 string concatenation techniques. 👍